viewer/src/utilities/networking/CogniteClient3dExtensions.ts

```diff
@@ -58,37 +55,40 @@ export class CogniteClient3dExtensions
     return response.data;
   }
 
-  public async getModelUrl(modelIdentifier: { modelRevision: IdEither; format: File3dFormat }): Promise<string> {
-    const { modelRevision, format } = modelIdentifier;
-    const outputs = await this.getOutputs(modelRevision, [format]);
+  public async getModelUrl(modelIdentififer: {
+    modelId: number;
+    revisionId: number;
+    format: File3dFormat | string;
+  }): Promise<string> {
+    const { modelId, revisionId, format } = modelIdentififer;
+    const outputs = await this.getOutputs({ modelId, revisionId, format });
     const mostRecentOutput = outputs.findMostRecentOutput(format);
     if (!mostRecentOutput) {
       throw new Error(
-        `Model '${modelRevision}' is not compatible with this version of Reveal. If this model works with a previous version of Reveal it must be reconverted to support this version.`
+        `Model '${modelId}/${revisionId}' is not compatible with this version of Reveal. If this model works with a previous version of Reveal it must be reconverted to support this version.`
       );
     }
     const blobId = mostRecentOutput.blobId;
     return `${this.client.getBaseUrl()}${this.buildBlobRequestPath(blobId)}`;
   }
 
-  public async getOutputs(modelRevisionId: IdEither, formats?: (File3dFormat | string)[]): Promise<Model3DOutputList> {
-    const url = `/api/playground/projects/${this.client.project}/3d/v2/outputs`;
-    const request: OutputsRequest = {
-      models: [modelRevisionId],
-      formats
-    };
-    const response = await this.client.post<ItemsResponse<{ model: IdEither; outputs: BlobOutputMetadata[] }>>(url, {
-      data: request
-    });
+  public async getOutputs(modelIdentifier: {
+    modelId: number;
+    revisionId: number;
+    format?: File3dFormat | string;
+  }): Promise<Model3DOutputList> {
+    const { modelId, revisionId, format } = modelIdentifier;
+    const url = `/api/v1/projects/${this.client.project}/3d/models/${modelId}/revisions/${revisionId}/outputs`;
```

Comment:
@larsmoa why don't we have it in public API? I mean, I don't see anything like [outputs](https://docs.cognite.com/api/v1/#tag/3D-Models) in the API reference.

Reply:
It's actually on its way, just hasn't made it to the doc yet. We should move the stuff we implement here to @cognite/sdk